### PR TITLE
fix typos and links in various documentation files

### PR DESCRIPTION
**Pull request by RisGar**
## Proposed Changes (include Screenshots if possible)

- Fixed typographical errors in documentation Markdown files
- Fixed minor grammatical issues in documentation Markdown files
- Replaced wrong links for Rust and actix-web in deployment README
- Replaced stylesheet link in map README with the "raw" link that Maputnik accepts
- Updated the WSL installation link in dependencies documentation
- Removed duplicate paragraphs

## How to test this PR

1. Look at the Markdown preview in GitHub

## How has this been tested?

- No code has been changed
- Markdown links to renamed file have been updated

## Checklist

- [X] I have updated the documentation / ~~No need to update the documentation~~
- [X] I have run the linter

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -123,10 +123,10 @@ data
 
 ## Compilation process
 
-The data compilation is made of indiviual processing steps, where each step adds new or modifies the current data. The
-basic structure of the data however stays the same from the beginning on and is specified in `data-format_*.yaml`.
+The data compilation is made of individual processing steps, where each step adds new or modifies the current data. The
+basic structure of the data, however, stays the same from the beginning on and is specified in `data-format_*.yaml`.
```
Comment (CommanderStorm) [inline data/README.md:127]: PSA: "However" cannot live in the middle of a sentence, only the first word.

I have also added a bit more context below

```suggestion
The data compilation is made of individual processing steps, where each step adds new or modifies the current data.
The final structure of the data, is specified in `data-format_*.yaml`.
Some work is underway to ensure that this format is actually being followed via simplifying the data backend and migrating the database server from managing a json blob to "real" tables. This is not done yet.
```

File: deployment/README.md [hunk 2]
```diff
@@ -41,10 +41,10 @@ On every of these nodes we run the following supporting services:
 
 We run the following components at these paths:
 
-- `/` [webclient](../webclient) written in [`Vue`]/[`nuxt-4`](https://nuxt.dev)
-- `/api/` [server](../server) written in [`Rust`](https://nuxt.dev)/[`actix-web`](https://nuxt.dev)
-- `/tiles/` [`marvin`] tiles from OSM
-- `/map/` [`tileserver-gl`] rendering tiles from OSM
+- `/` [webclient](../webclient) written in [`Vue`](https://vuejs.org/)/[`nuxt-4`](https://nuxt.dev)
+- `/api/` [server](../server) written in [`Rust`](https://www.rust-lang.org/)/[`actix-web`](https://actix.rs/)
+- `/tiles/` [`martin`](https://martin.maplibre.org/) tiles from OSM
```
Comment (CommanderStorm) [inline deployment/README.md:46]: Good catch xD

**Review by CommanderStorm — APPROVED** [review]
Good stuff ✨
Tanks

File: data/README.md (hunk 1 above)
Comment (RisGar) [reply data/README.md]: image
;)

Comment (CommanderStorm) [reply data/README.md]: hmm...
Okay, fair 😅

**Comment by CommanderStorm** [comment]
(idk why I did not merge earlier)

**Pull request merged**